### ISSUE TITLE
Exerciser highlight error

### ISSUE THREAD

**Issue by acoroleu-tempus** [jsonata-js/jsonata-exerciser#18]
While trying this expression from the docs in the exerciser:
Account.Order.Product { `Product Name`: {"Price": Price, "Qty": Quantity} }
The highlight seems to break for Price. On the other hand this expression does not break the highlight:
Account.Order.Product { `Product Name`: {'Price': Price, "Qty": Quantity} }

Am I doing something wrong? Any plans to fix it?

Thanks

**Comment by acoroleu-tempus** [comment]
Any updates on this? Thanks

**Comment by andrew-coleman** [comment]
No, but let me paste here the response I made when you raised this in the JSONata repo:

> The exerciser uses Monaco for the editor and supplies a [config file](https://github.com/jsonata-js/jsonata-exerciser/blob/master/src/jsonataMode.js) which instructs the editor how to highlight the tokens. You could try fiddling with this to see if you can get any better results than I've managed.

This is not something I am going to get to any time soon, but I'm happy to look at a PR if you improve how it renders.

Thanks.

**Comment by DevDimov** [comment]
Hi @acoroleu-tempus, it looks like the issue has been resolved. Have you had any other highlight errors in the meantime?

![Screenshot 2024-05-31 125420](https://github.com/jsonata-js/jsonata-exerciser/assets/77745022/f357164a-d799-4662-a448-c43296387840)
